src/Supplier.php

```diff
@@ -58,6 +58,11 @@ public static function getTypeName($nb = 0)
         return _n('Supplier', 'Suppliers', $nb);
     }
 
+   public function post_getEmpty()
+   {
+      $this->fields['is_active'] = 1;
```

Comment:
Suppliers should be inactive by default it seems based on the current DB schema.